### DISCUSSION TITLE
Does this work with the GPU version?

### DISCUSSION THREAD

**Discussion by sagarspatil** [neilenns/node-deepstackai-trigger#405] [Q&A]
Hello,

First of all, thank you for creating this. I really appreciate it. 

When I change the deepstack docker image to GPU, it stops analysing images. Does your software work with GPU docker image?

**Comment by neilenns** [comment]
Are there any errors showing in the logs?

I'm 99% sure the GPU version of Deepstack has some sort of silly product key requirement that you have to get through before it'll work.

**Comment by neilenns** [comment]
See https://deepstack.cc/install/docker down near the bottom for activation instructions.
